engines/base/src/main/resources/application.yaml

```diff
@@ -0,0 +1,64 @@
+---
+spring:
+  servlet:
+    multipart:
+      max-file-size: 8192MB
+      max-request-size: 8192MB
+  activemq:
+    broker-url: ${ACTIVEMQ_URL:nio://localhost:61616}?jms.watchTopicAdvisories=false
+    user: ${ACTIVEMQ_USER:admin}
+    password: ${ACTIVEMQ_PASSWORD:admin}
+    pool:
+      enabled: true
+      max-connections: 20
+  jackson:
+    default-property-inclusion: non_empty
+
+activemq:
+  url: ${ACTIVEMQ_URL:false}
+
+server:
+  port: ${SERVER_PORT:8090}
+  error:
+    include-message: ALWAYS
+
+# Historic values (AVOID) - Exist to help with transition to newer versions
+transformer-routes-path: ${TRANSFORMER_ROUTES_FILE_LOCATION:transformer-pipelines.json}
+
+logging:
+  level:
+    org.alfresco.transform.common.TransformerDebug: debug
+
+filestore-url: ${FILE_STORE_URL:http://localhost:8099/alfresco/api/-default-/private/sfs/versions/1/file}
+
+transform:
+  core:
+    version: @project.version@
```

Comment:
My editor flags this as an error: (may be false positive?)

`Plain value cannot start with reserved character`

Reply:
Strange. Mine does not and it is correct.